test/controllers/welcome_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class WelcomeControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get root_url
```

Comment:
should be get :index, down the road we could possibly change the root url and this would no longer then test the welcome index action